docs/20_smart-contracts/10_getting-started/20_accounts/index.md

```diff
@@ -4,37 +4,37 @@ title: Account Basics
 
 ## What is an account?
 
-An account on the EOS Network is a digital container that holds a variety of information like EOS tokens that the account owns, resources at its disposal, account control permission structure, and various other things we will get into later in this guide. Accounts can also hold a smart contract. 
+On the EOS Network an account is a digital container. The digital container holds a variety of information such as EOS tokens the account owns, resources at its disposal, and the account permissions structure. Accounts can also hold a smart contract, tables, indexes, and one singleton. An account is the key component that enables access and control of your blockchain data and to execute transactions.

```

Comment:
"tables, indexes, and one singleton" the reader should not be introduced to the words tables, indexes, or singletons in this guide as it comes before the smart contract guide and isn't explained in this guide. It opens a loop that this guide never closes.